lib/discourse_theme/scaffold.rb

```diff
@@ -2,159 +2,94 @@
+               exception: true
+        FileUtils.rm_rf(SKELETON_DIR)
+        # Store the local copy for offline use
+        FileUtils.cp_r(tmp, SKELETON_DIR)
+
+        FileUtils.cp_r(SKELETON_DIR, dir)
+      elsif Dir.exist?(SKELETON_DIR)
+        puts "⚠️ No internet connection detected, using the local copy of discourse-plugin-skeleton"
+        FileUtils.cp_r(SKELETON_DIR, dir)
+      else
+        raise "🛑 Couldn't download discourse-plugin-skeleton"
+      end
 
+      Dir.chdir dir do
+        author = UI.ask("Who is authoring the theme?", default: "Discourse").to_s.strip
         description = UI.ask("How would you describe this theme?").to_s.strip
 
-        about_template = ABOUT_JSON.dup
-        about_template[:name] = name
-        if is_component
-          about_template[:component] = true
-        else
-          about_template[:color_schemes] = {}
+        about = JSON.parse(File.read("about.json"))
+        about["name"] = name
+        about["authors"] = author
+        if !is_component
+          about.delete("component")
+          about["color_schemes"] = {}
+        end
+        File.write("about.json", JSON.pretty_generate(about))
+
+        if author != "Discourse"
+          license = File.read("LICENSE")
+          license.sub!(/^(Copyright\s\(c\))\s(.+)/, "\\1 #{author}")
+          File.write("LICENSE", license)
         end
 
-        encoded_name = name.downcase.gsub(/[^a-zA-Z0-9_-]+/, "_")
+        readme = File.read("README.md")
+        readme.sub!("**Theme Name**", "**#{name}**")
+        File.write("README.md", readme)
 
-        write("about.json", JSON.pretty_generate(about_template))
-        write("HELP", HELP)
-        write("LICENSE", LICENSE.sub("#YEAR", "#{Date.today.year}").sub("#AUTHOR", author))
-        write(".eslintrc", ESLINT_RC)
-        write(".gitignore", GIT_IGNORE)
-        write(".template-lintrc.js", TEMPLATE_LINT_RC)
-        write("package.json", PACKAGE_JSON.sub("#AUTHOR", author))
-        write("settings.yml", SETTINGS_YML)
-        write("common/common.scss", "")
-        write("javascripts/discourse/api-initializers/#{encoded_name}.js", API_INITIALIZER)
-        write("locales/en.yml", EN_YML.sub("#DESCRIPTION", description))
+        encoded_name = name.downcase.gsub(/[^a-zA-Z0-9_-]+/, "-")
+        FileUtils.mv(
+          "javascripts/discourse/api-initializers/todo.js",
+          "javascripts/discourse/api-initializers/#{encoded_name}.js",
+        )
+
+        i18n = YAML.safe_load(File.read("locales/en.yml"))
+        i18n["en"]["theme_metadata"]["description"] = description
+        File.write("locales/en.yml", YAML.safe_dump(i18n).sub(/\A---\n/, ""))
 
         UI.info "Initializing git repo"
-        puts `git init && git symbolic-ref HEAD refs/heads/main`
+        FileUtils.rm_rf(".git")
+        system "git", "init", exception: true
+        system "git", "symbolic-ref", "HEAD", "refs/heads/main", exception: true
+        system "git", "add", "-A", exception: true
```

Comment:
How about only committing the bare essential files, and then people can add the rest themselves once they have real content.

IMO the only things we should commit are the config files in the root directory, and maybe the locale file with the description. Everything else in the skeleton is just a placeholder and doesn't really make sense to commit.

Also... wondering if we should `rm **/*.gitkeep` to keep things even cleaner 🤔